Vagrantfile

```diff
@@ -111,10 +111,17 @@ Vagrant.configure("2") do |config|
 
 	# Ensure that WordPress can install/update plugins, themes and core
 	if vagrant_version >= "1.3.0"
-		config.vm.synced_folder ".", "/vagrant", :mount_options => [ "dmode=777,fmode=777" ]
-		CONF["synced_folders"].each do |from, to|
-			config.vm.synced_folder from, to, :mount_options => [ "dmode=777,fmode=777" ]
-		end if CONF["synced_folders"]
+		if CONF['nfs'] == true
+			config.vm.synced_folder ".", "/vagrant", :nfs => { :mount_options => ["dmode=777","fmode=777"] }
```

Comment:
This is actually not doing what you want, as far as I can see. `:nfs` is a bool option, so the hash gets coerced to true, but the options get lost. You can verify this by using `:nfs => true, :mount_options => ...`, which gives an error.

Fixing it up now.